GTG/gtk/browser/tag_context_menu.py

```diff
@@ -68,7 +68,7 @@ def __build_menu(self):
                 self.mi_del.connect('activate', self.on_mi_del_activate)
             else:
                 self.mi_del_tag = Gtk.MenuItem()
-                self.mi_del_tag.set_label(_("Delete Tag"))
+                self.mi_del_tag.set_label(_("Delete"))
```

Comment:
I'm not convinced removing context from the string is warranted, but I didn't go looking up if it's clear enough anyways or something. Unifying strings for minimizing translations shouldn't ever be a concern - in fact, some languages may want to translate a generic "Delete" differently than if it were to refer to a specific thing like a tag.
That said, maybe the specification of "Tag" is unnecessary here separate from the translation misconcerns anyways.

Reply:
Yeah, since it's in a right-click menu, it seems to me that the context could not be mistaken for the user...